src/lib/services.js

```diff
@@ -140,6 +140,16 @@ export function getUserEnrollments({
     .then(({ data }) => data);
 }
 
+export function getUserEnrollmentsBis() {
+  return httpClient
+    .get(`${BACK_HOST}/api/enrollments/user`, {
```

Comment:
sorting should be static and explicit here, we should not rely on the default backend value

Reply:
ok. I didn't read the backend PR. Forget about this comment.